x-pack/plugins/actions/server/builtin_action_types/pagerduty.ts

```diff
@@ -86,7 +86,7 @@ const ParamsSchema = schema.object(
 
 function validateParams(paramsObject: unknown): string | void {
   const { timestamp, eventAction, dedupKey } = paramsObject as ActionParamsType;
-  if (timestamp != null) {
+  if (timestamp != null && timestamp.length > 0) {
```

Comment:
Because the input field defaults to an empty string, the optional timestamp field on the PagerDuty connector params shows up as an empty string instead of null when not filled in. This just updates the validation to skip validation if timestamp is an empty string.